### ISSUE TITLE
Exclude Category from Largo Recent Posts Widget

### ISSUE THREAD

**Issue by Ogmeister** [INN/Largo#423]
Hello - 

I have one category that I wish to exclude from the recent posts. What is the best way to do that?

**Comment by aschweigert** [comment]
You could probably use the "limit by custom taxonomy" option and use "category" as the taxonomy and then whichever category you want to exclude, use the category ID preceded by a minus sign, (i.e. - "-12" where the category ID for the category you'd like to exclude is no. 12). I'm not 100% sure that will work but that would be another thing to experiment with. If that doesn't work we'll add this a feature request for a future release.

**Comment by Ogmeister** [comment]
What is the syntax for that? I've tried -#, cat=-#, category=-#, and variations with spaces. Everything I've tried "breaks" the widget - the output is "You don't have any recent articles."

**Comment by aschweigert** [comment]
Actually, sorry, it uses the slug (basically it's just passing a value to WP_Query using tax_query (http://codex.wordpress.org/Class_Reference/WP_Query#Taxonomy_Parameters) and I don't think the way we have it written right now would allow you to use the NOT IN operator so we might be out of luck. We'll look at this and see if we can figure out something in our next iteration.

**Comment by Ogmeister** [comment]
Hmm... Here's the issue - I want to include obituaries on my news site, but I don't want them on the front page, only on their page. Do you have any suggestions for how I can handle that? A simple recent posts plugin would be hard to configure to skip the posts in the Featured Posts section.

**Comment by aschweigert** [comment]
It's a bit hard to answer that without having a little more context regarding how your homepage is laid out. There's an option we add in the Appearance > Theme Options > Layout menu to exclude categories from the homepage loop, but that's only applicable if you're using the "river" homepage layout for the bottom section of the homepage. It sounds like that's not what you're after though so the recent posts widget would be the best way to go, it's just that we haven't yet accounted for the ability to exclude a category from this widget (although that's something it would definitely make sense to try to do). Of course you could create a custom widget in your child theme and modify the widget's behavior but that would depend a bit on your coding ability.

**Comment by Ogmeister** [comment]
The site is www.FourStatesNews.us, if you'd like to take a look. It's the Blargo fork/variation of Largo. There is an option to exclude categories, but I just tested it and it doesn't function when the Largo Recent Posts Widget is activated.

**Comment by aschweigert** [comment]
Ah, yeah I'd recommend against using Blargo at this point unless you need something specific that it includes that our latest release does not. We changed so many things in the 0.4 release that I wouldn't even be able to give you much advice on where to start at this point because their fork is so far out of date.

**Comment by Ogmeister** [comment]
Well, I thought I was getting something geared toward ads, but I had to tweak a few things to get the ads below the Featured Stories area. I really like the look I have; what would switching to Largo do, and how painful would it be? I do intend to used the Monster Slot ad soon - is that available on Largo?

**Comment by aschweigert** [comment]
Their terminology is kind of odd so I'm not sure what they mean by "monster" but...probably? We mostly work with sites that are using Google DFP and we just released a plugin to streamline that process (https://github.com/INN/DoubleClick-for-WordPress). I'm not terribly familiar with broadstreet and what they were doing seemed fine at the time but my main concern at this point is just how far behind they are so you wouldn't be able to get much in the way of future updates to largo (and there have been a lot) unless they decide to do some work to update their fork of it.

**Comment by Ogmeister** [comment]
Well, I'm happy with my site, and the new layout, which is a week or two old. I've gotten lots of compliments on it. My main concern is how to re-implement obituaries while keeping them off the front page. :-)

**Comment by Ogmeister** [comment]
The new Largo stuff is beautiful, but kind of overshoots what I'm doing, I think. LOL

**Comment by aschweigert** [comment]
Well, I will say that we spent a lot of time on legacy support so all of the layout stuff from the previous version is still available. It's just that if what you need is something we include in a future update you're going to be out of luck unless the blargo folks update their fork and I'm not sure what their plans are on that front.

**Comment by Ogmeister** [comment]
I'll certainly keep it in mind. 

When I was including obituaries last fall, it was quickly becoming an attraction to my site. It didn't take long to grow to at least 10% of the traffic. The ambition of including obits from the whole region was taking a lot of time, though. I have things a little more streamlined now, so if you can think of a way to exclude them from the main page, I'd be very grateful.

**Comment by aschweigert** [comment]
With the version that blargo is forked from there wouldn't be an option other than the "exclude from homepage" option in the theme settings.

**Comment by Ogmeister** [comment]
Does the current version of Largo have that option using the recent posts widget?

**Comment by aschweigert** [comment]
It does not, but what I'm saying is that even if we add it in a future release you're not going to be able to use it unless you're tracking the version that includes the latest updates.

**Comment by Ogmeister** [comment]
I'll keep an eye out, then. I've got amateur web developer fatigue right now. LOL

**Comment by Ogmeister** [comment]
In my old setup, I had a category called Front Page, which I used for the same effect I'm looking for now, but I'm trying to start using only one category per post. Could I make a new category in Post Prominence, which seem to be "invisible" tags, and use Limit to Tags?

**Comment by aschweigert** [comment]
You wouldn't use limit to tags, you'd use limit to taxonomy/term, but yeah that could work.

**Comment by Ogmeister** [comment]
First, at Posts>Post Prominence, I created a new category called Front Page, with the slug front-page. Then, on the Largo Recent Posts widge, under Taxonomy I entered prominence, and under Term I entered front-page. Trial and error showed that I couldn't use Front Page or the term ID number. 

It's working great - I don't have a second category for each post, the tag should be invisible, and I can post anything I want without putting it on the front page, including obits. 

Thank you so much for the help!
